src/screens/Home/index.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import Footer from '../../components/Footer';
 import Navbar from '../../components/Navbar';
 import Sidebar from '../../components/Sidebar';
+import HomePageSearch from '../../components/Search';
```

Comment:
Lets just name it by element, call it `SearchTab`.

Reply:
Just a reminder, both file name and component name should be the same.

Reply:
Just to clarify, Is this what you want me to modify?
```suggestion
import HomePageSearch from '../../components/SearchTab';
```
And which name should be the same, do I have to rename 'HomePageSearch' to be 'SearchTab'. And how about the name 'HomePage' in https://github.com/Cobda/cobda-web/pull/5#discussion_r563300259

Reply:
To clarify, so you want me to change className to be `.Homepage`. And inside, there's a `&--search-tab` element, right?